app/Models/Project.php

```diff
@@ -371,11 +371,11 @@ public function getEntitySubsteps(): array
     }
 
     /**
-     * Get the project substeps for consultants.
+     * Get the project substeps for community-members.
```

Comment:
should this be participants. It seems that you refer to community members as participants in relation to the Project.